docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Asynchronous webhooks
+---
+
+## Architecture
+
+Asynchronous webhooks can be used for reacting to the events emitted by the Saleor. Worker instance will send event payload via the chosen protocol. Your app should acknowledge that payload was received. The result of app operations is not sent back to the Saleor. If you need to send data back to the user (for example, calculated shipping cost), you should use [synchronous webhook](developer/extending/apps/synchronous-webhooks.mdx).
```

Comment:
We could mention here that webhooks as a response accept 2XX codes. If Saleor gets different than 2XX, we will retry to send webhook once again (max retries 5)

Reply:
Do we retry if we get a 4xx? Should we?

Reply:
What do we do if we get a 3xx redirect?

Reply:
I believe we should de-activate webhook when we get 4XX (we have task for this)

Reply:
@korycins Retry already mentioned https://github.com/mirumee/saleor-docs/pull/284/files#diff-14342a206c0a26821cb1f77b7d62ddf8dbd53ab2d4ac1022df54bb5f1ac6c5fdR38

Reply:
IMO https section is better place than architecture for such info